### PR TITLE
Remove mongo intersphinx mapping.

### PR DESCRIPTION
**Pull request by manthey**
It is broken and prevents the docs from being generated.

### CODE REVIEW

**Comment by manthey** [comment]
Our codecov reporting is still broken, but this fixes the docs build.

**Review by zachmullen — APPROVED** [review]

File: docs/deployment-alternatives.rst [hunk 1]
```diff
@@ -80,7 +80,7 @@ And under the containing ``server`` block, make sure to add the following rule:
 
 Ansible
 -------
-:doc:`installation` provides a complete installation workflow, which depends primarily on the
+The Ansible deployment provides a complete installation workflow which depends primarily on the
```
Comment (brianhelba) [inline docs/deployment-alternatives.rst:83]: Is it possible to use another type of intra-doc link here? I'd like to funnel people to the installation page as much as possible.

Comment (manthey) [reply docs/deployment-alternatives.rst:83]: @brianhelba Before it had an erroneous link to mongo's documentation.  I think wrong links are worse than no links.  I asked where this should have pointed to on slack, but no one responded. It isn't clear to me where it was supposed to reference.

Comment (brianhelba) [reply docs/deployment-alternatives.rst:83]: Ah. This should reference https://github.com/girder/girder/blob/master/docs/deployment.rst

**Pull request merged**